PowerEditor/src/resource.h

```diff
@@ -431,6 +431,7 @@
     //#define IDC_KEY_END                        (IDCMD+7)
     //#define IDC_KEY_SELECT_2_HOME            (IDCMD+8)
     //#define IDC_KEY_SELECT_2_END            (IDCMD+9)
+    #define IDC_REFRESH_WORK_DIR           (IDCMD+10)

```

Comment:
It should be rather `NPPM_INTERNAL_REFRESHWORKDIR`.

Reply:
Thanks for explanation. I'm not very familiar with this part of the app.